website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -48,7 +83,7 @@ export default function ConsulOnKubernetesPage() {
                 text: 'Try It Now',
                 url: '#TODO',
               },
-              image: require('./images/multi-platform.svg'),
+              image: require('./images/features/multi-platform.svg'),
```

Comment:
I re-organized the local images folder a bit

Reply:
Good call 👌 👍